extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -1643,5 +1668,16 @@ namespace Microsoft.Windows.DevHome.SDK
         IQuickStartProjectGenerationOperation CreateProjectGenerationOperation(String prompt, Windows.Storage.StorageFolder outputFolder);
     }
 
+    // Extensions can implement this provider to provide a way to
+    // create a project based on a prompt as part of the Dev Home 
+    // quick start project feature.
+    [experimental]
+    [contract(Microsoft.Windows.DevHome.SDK.DevHomeContract, 6)]
+    interface IQuickStartProjectProvider2 requires IQuickStartProjectProvider
+    {
+        // Pass additional flag to allow the ability to ignore prompt guidance / errors
```

Comment:
same here for this comment.

Reply:
Oops! I left some old comments up, good catch. Should be revised now.